blueflood-http/src/test/java/com/rackspacecloud/blueflood/inputs/handlers/HttpEventsIngestionHandlerTest.java

```diff
@@ -33,7 +38,7 @@
 import static org.mockito.Mockito.*;
 import static junit.framework.Assert.*;
 
-public class HttpEventsIngestionHandlerTest {
+public class HttpEventsIngestionHandlerTest extends BaseHandlerTest {
```

Comment:
I just noticed this... If you name a class *Test, the junit test runner will try to run it. If this is meant to be a base class, and not a test class, can we name it HandlerTestsBase ?